hm_21/task_4

```diff
@@ -0,0 +1,7 @@
+import sqlite3
+
+query = ("SELECT age, COUNT(*) AS users_amount FROM users GROUP BY age ORDER BY users_amount desc")
```

Comment:
Here you should implement multiple ordering: `... ORDER BY users_amount desc, age`